package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "femmecubator-api-repo",
+  "version": "1.0.0",
+  "description": "Node.js backend for Femmecubator",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
- Update scripts section:
        - “start”: “node ./server/app.js”  // we will update this in the future to point to a build version
        - “start-debug”: “nodeman ./server/app.js”
        - “test”: “jest —coverage —coverageDirectory=output/coverage/jest —verbose —detectOpenHandles —forceExit”
        - “coverage”: “jest —coverage —coverageDirectory=output/coverage/jest”
        - “lint”: “eslint -f html . -o output/eslint/eslint-report.html —cache —cache-location output/eslint/“
        - “fix:eslint”: “eslint —fix \”**/*.*\””
        - “fix:prettier”: “prettier —write \”**/*.*\””
        - “fix”: “run-p fix:*”

Reply:
done in commit # https://github.com/cfernando415/femmecubator-api-repo/pull/1/commits/8d0e3e9027b8e266c088d74fb6d83d47e3badd75